jwt_proxy/api.py

```diff
@@ -4,7 +4,7 @@
 import json
 
 blueprint = Blueprint('auth', __name__)
-SUPPORTED_METHODS = ('GET', 'POST', 'PUT', 'DELETE')
+SUPPORTED_METHODS = ('GET', 'POST', 'PUT', 'DELETE', 'OPTIONS')
```

Comment:
Can you put this in a separate PR (no need for review), or commit directly to `main`?

Changing CORS to enable a `public` SoF client is an important enough change I think merits its own commit :)

Reply:
Done: https://github.com/uwcirg/jwt-proxy/pull/5